index.html

```diff
@@ -7,8 +7,30 @@
   <body>
     <div class ="main">
       <header>
+        <span>Let's Help Ourselves</span>
       </header>
+      <section class="container">
+          <div class="home-msg">
+            <p class="line1">The First Equity Based Crowdfunding and an Integrated E-Commerce Company</p>
+            <p class="line2">for</p>
+            <p class="line3">Non Resident Indian Returnees</p>
+          </div>
+      </section>
       <footer>
+        <div class="footer-left">
+        </div>
+        <div class="footer_mid">
+            <ul>
+                <li class="lists"><a href="" class="active">Home</a></li>
+                <li class="lists"><a href="about.html">About Us</a></li>
+                <li class="lists"><a href="founder.html">Founder</a></li>
+                <li class="lists"><a href="businessLogics.html">Business Logics</a></li>
+                <li class="lists"><a href="when.html">When</a></li>
+                <li class="lists"><a href="contactUs.html">Write to us</a></li>
```

Comment:
The class lists can be added to ul and styled accordingly, so that the class wouldn't have to be added to each li.